hrms/hr/doctype/leave_application/leave_application.py

```diff
@@ -1034,6 +1044,23 @@ def get_holidays(employee, from_date, to_date, holiday_list=None):
 
 	return holidays
 
+@frappe.whitelist()
+def get_holidays_list(employee, from_date, to_date, holiday_list=None):
```

Comment:
`get_holidays_for_employee` is already exists under `hrms/utils.py` no need for a new function.